### PR TITLE
Adds capitalize helper

### PR DESCRIPTION
**Pull request by vikram7**
This PR adds the `capitalize` helper per the issue here: https://github.com/DockYard/ember-composable-helpers/issues/71

### CODE REVIEW

File: addon/helpers/capitalize.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+const {
+  Helper: { helper },
+  String
```
Comment (poteto) [inline addon/helpers/capitalize.js]: Can destructure `capitalize` here:

```js
const {
  String: { capitalize }
} = Ember;
```

File: addon/helpers/capitalize.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+} = Ember;
+
+export function capitalize([string]) {
+  return String.capitalize(string);
```
Comment (poteto) [inline addon/helpers/capitalize.js]: We should add a guard for `string` – if it isn't actually a `String`, the Ember method will explode. Please also add a test case for when the argument is undefined

**Comment by poteto** [comment]
Please also add an export to https://github.com/DockYard/ember-composable-helpers/blob/master/addon%2Findex.js

**Comment by poteto** [comment]
Thanks for the PR!

File: addon/helpers/capitalize.js (hunk 1 above)
Comment (vikram7) [inline addon/helpers/capitalize.js]: I had tried that earlier, but after running `ember test` I got some `Duplication declaration "capitalize"` errors:

```
Build failed.
File: modules/ember-composable-helpers/helpers/capitalize.js
modules/ember-composable-helpers/helpers/capitalize.js: Line 7: Duplicate declaration "capitalize"
TypeError: modules/ember-composable-helpers/helpers/capitalize.js: Line 7: Duplicate declaration "capitalize"
   5 | } = Ember;
   6 |
>  7 | export function capitalize([string]) {
     |                 ^
   8 |   return capitalize(string);
   9 | }
  10 |
```
Any suggestions on how to destructure here?

File: addon/helpers/capitalize.js (hunk 2 above)
Comment (vikram7) [inline addon/helpers/capitalize.js]: :+1:

File: addon/helpers/capitalize.js (hunk 1 above)
Comment (poteto) [inline addon/helpers/capitalize.js]: Try String: { capitalize: _capitalize }

File: addon/helpers/capitalize.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+const {
+  Helper: { helper },
+  String: { capitalize: _capitalize }
+} = Ember;
+
+export function capitalize([string]) {
+  if (string) {
```
Comment (poteto) [inline addon/helpers/capitalize.js]: This could be

```js
const { typeOf } = Ember;

...

string = string || '';

return _capitalize(string);
```

Comment (vikram7) [inline addon/helpers/capitalize.js]: :+1:

**Comment by poteto** [comment]
One last thing, please update the readme:

```md
#### `capitalize`
Capitalizes a string using `Ember.String.capitalize`.

\```hbs
{{capitalize "hello jim bob"}}
{{capitalize fullName}}
\```

**[⬆️ back to top](#available-helpers)**
```

**Comment by poteto** [comment]
@homu r+

**Pull request merged**